gcc/rust/expand/rust-macro-expand.h

```diff
@@ -180,8 +202,9 @@ class SubstitutionScope
     auto it = current_map.find (fragment.fragment_ident);
 
     if (it == current_map.end ())
-      current_map.insert ({fragment.fragment_ident,
-			   MatchedFragmentContainer::metavar (fragment)});
+      current_map.emplace (fragment.fragment_ident,
+			   std::move (
+			     MatchedFragmentContainer::metavar (fragment)));
```

Comment:
```suggestion
			     MatchedFragmentContainer::metavar (fragment));
```
oh, that `move` is unnecessary and causes clang to emit a warning - I'll merge the PR once this is removed

Reply:
This should be fixed, just waiting for the tests to run